### PR TITLE
Add commrnt

### PR DESCRIPTION
**Pull request by NP-SON-250**
csjgd

### CODE REVIEW

**Comment by NP-SON-250** [comment]
## What this Pull Request Does

Add Comment Side
Add Documentation Codes

File: src/controller/commentController.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import Comments from "../model/commentModel";
+import Posts from "../model/postModel";
+import Users from "../model/userModel";
+
+export const addComment = async (req, res) => {
+  try {
+    const {Post_Id} = req.params;
```
Comment (mansurissa) [inline src/controller/commentController.js]: Please use camelCase like `postId` for this case

File: src/controller/commentController.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+};
+
+
+
+
+
+export const getAllComments = async (req, res) => {
+    try {
+      const commentView = await Comments.find()
+        // .populate("User_ID", "First_Name Last_Name Email Profile")
```
Comment (mansurissa) [inline src/controller/commentController.js]: Delete commented code

File: src/index.js [hunk 3]
```diff
@@ -20,21 +23,21 @@ mongoose
...
     openapi : '3.0.0',
     info : {
-      title: 'Node JS And Mongo DB Klab Blog Api Project',
+      title: 'Klab Blog API Node JS',
       version: '1.0.0'
     },
     servers:[
       {
-        url: 'https://klab-blog-api.onrender.com/'
+        // url: 'https://klab-blog-api.onrender.com/'
```
Comment (mansurissa) [inline src/index.js]: Remove commented code

File: src/middleware/commentAuthontication.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    const logedUser = await Users.findById(decoded.id);
+
+    if (!logedUser) {
+      res.status(403).json({
+        status: "403",
+        message: "Token has Expired Please login Again",
+      });
+    }
+
+      req.Users = logedUser;
```
Comment (mansurissa) [inline src/middleware/commentAuthontication.js]: You can name this to something like `req.loggedInUser = loggedInUser`

**Review by mansurissa — CHANGES_REQUESTED** [review]
Good job, just work on simple changes asked and re-request review

File: src/controller/commentController.js (hunk 1 above)
Comment (NP-SON-250) [reply src/controller/commentController.js]: Resolved camelCase naming

File: src/index.js (hunk 3 above)
Comment (NP-SON-250) [reply src/index.js]: Resolved case of commented codes

File: src/middleware/commentAuthontication.js (hunk 4 above)
Comment (NP-SON-250) [reply src/middleware/commentAuthontication.js]: Resolved naming from req.User = logedInUser to req.loggedInUser = loggedInUser

File: src/controller/commentController.js (hunk 2 above)
Comment (NP-SON-250) [reply src/controller/commentController.js]: Removed commented code

**Review by mansurissa — APPROVED** [review]
LGTM

**Pull request merged**